docs/spec/src/types.ts

```diff
@@ -0,0 +1,20 @@
+import {
```

Comment:
You can upgrade `@jupiterone/integration-sdk-core@6.19.0` (and other packages) - then you can import these types and the `toImplementSpec` matcher.